client/src/app/+videos/+video-watch/video-watch.component.html

```diff
@@ -54,7 +54,12 @@ <h1 class="video-info-name">{{ video.name }}</h1>
             <div class="video-info-date-views">
               <ng-container i18n>Published <my-date-toggle [date]="video.publishedAt"></my-date-toggle></ng-container>
 
-              <span i18n class="views">
+              <span i18n i18n-title
+                    title="{{
+                      (video.views >= 1000)
+                      ? (video.views + ' ' + (video.isLive ? 'views' : 'viewers'))
+                      : ''}}"
```

Comment:
This is too much logic for templates, let alone translation templates. I suggest you put everything aside in a getter in the component's controller.

Reply:
I make the change, thks